index.html

```diff
@@ -209,6 +209,57 @@ <h2>
             <button type="submit">See Project</button>
         </div>
     </section>
+
+    <section class="contact-container">
+        
+        <div class="contact-intro">
+            <div class="contact-header">
+                <h1 >CONTACT US</h1>
+            </div>
+            <div class="contact-desc">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            
+        </div>
+
+        <form action="#" method="post" class="contact-form">
+            <div class="contact-inbox">
```

Comment:
- [ ] Please make sure to add the [formspree service](https://formspree.io/html) to collect user data, as per requirement.